schema/spacewalk/upgrade/susemanager-schema-4.3.3-to-susemanager-schema-4.3.4/002-use_almalinux_mirror.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE rhnContentSource SET source_url = 'https://mirrors.almalinux.org/mirrorlist/8/baseos' where label like 'External - AlmaLinux 8 (%)';
```

Comment:
Nitpick:

```suggestion
UPDATE rhnContentSource SET source_url = 'https://mirrors.almalinux.org/mirrorlist/8/baseos' WHERE label LIKE 'External - AlmaLinux 8 (%)';
```
(same for the other lines)

Reply:
Can you also do an two extra test for me?

InstanceA, using current master:
- Add all the channels for all architectures, not just `x86_64`
- Apply the SQL
- Try to sync all of them.

Instance B, using this PR:
- Add all the channels for all architectures, not just `x86_64`
- Try to sync all of them.

In the second case, I am 99.99% sure there won't be problems.

In the first one, I am less sure. Should, but... :-D